arcgis-ios-sdk-samples/Layers/Play a KML Tour/README.md

```diff
@@ -0,0 +1,48 @@
+# Play a KML Tour
+
+Play tours in KML files.
+
+![Screenshot](image1.png)
+
+## Use case
+
+KML, the file format used by Google Earth, supports creating tours, which can control the viewpoint of the scene, hide and show content, and play audio. Tours allow you to easily share tours of geographic locations, which can be augmented with rich multimedia. Runtime allows you to consume these tours using a simple API.
+
+## How to use the sample
+
+The sample will load the KMZ file automatically. When a tour is found, the Play button will be enabled. Use the Play and Pause buttons to control the tour. When you're ready to show the tour, use the reset button to return the tour to the unplayed state.
```

Comment:
I think the word `automatically` does not express how it's done. I would say `The sample will load the KMZ file that is hosted in ArcGIS Online`. Or, just `The sample will load the KMZ file`.